SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -178,7 +178,8 @@ let globalAttributesLock = NSLock()
 
 let splunkLibraryLoadTime = Date()
 var splunkRumInitializeCalledTime = Date()
-
+// swiftlint:disable missing_docs
+public var sessiontoken: String = ""
```

Comment:
I don't think the changes in this file are used anywhere

Reply:
removed